src/components/Book.js

```diff
@@ -1,5 +1,5 @@
 import React from 'react';
-import { string } from 'prop-types';
+import PropTypes from 'prop-types';
 
 export default function Book(props) {
   const { title, author } = props;
```

Comment:
- [ ] As per this [requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/bookstore/use_redux_in_react_components.md#:~:text=In%20your%20React%20component%20responsible%20for%20removing%20books%2C%20add%20a%20functionality%20for%20%22Remove%22%20button%20click), Please add functionality for "Remove" button click using `usedispatch` function.